packages/augur-sdk/src/state/logs/types.ts

```diff
@@ -244,6 +244,17 @@ export interface CurrentOrder extends ParsedOrderEventLog {
   open: number; // 0 == false, 1 == true. Can't index booleans
 }
 
+// from IExchangeCore.sol
+export interface CancelLog extends Log {
+  makerAddress: Address;
+  feeRecipientAddress: Address;
+  makerAssetData: Bytes32;
+  takerAssetData: Bytes32;
+  senderAddress: Address;
+  orderHash: Bytes32;
+  market?: Address; // only derived log CancelledOrders
```

Comment:
I don't think this should be optional property since there is an index for it.

Reply:
These are really two types, one for simple log one for derived. I'll split the type out.